pandas/core/dtypes/dtypes.py

```diff
@@ -17,6 +17,10 @@
 
 str_type = str
 
+# sentinel value used for the default value of ordered in the CategoricalDtype
+# constructor to detect when ordered=None is explicitly passed (GH 26403)
+sentinel = object()  # type: object
```

Comment:
Maybe `ordered_sentinel` to be more explicit

Reply:
done